x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -165,112 +152,11 @@ export const getMetadataRequestHandler = function (
         ),
       });
     } catch (error) {
-      return errorHandler(logger, response, error);
+      return errorHandler(response, error);
     }
   };
 };
 
-export async function getHostMetaData(
```

Comment:
Nice! It was no longer being used?

Reply:
Not at all! I was surprised to find this big chunk of code not used anywhere. The only usage of `getHostMetaData` was within `getHostData` at line 211